conf/service_wildcard.conf

```diff
@@ -3,14 +3,18 @@ server_name _;
 location / {
   access_by_lua_block {
     local wildcard_service = require("wildcard-service").new();
-    local servers = wildcard_service:get_apicast_servers();
+    local servers, err = wildcard_service:servers();
 
-    if #servers == 0 then
+    if err then
+      ngx.log(ngx.WARN, 'could not find servers: ', err)
+    end
+
+    if not servers or #servers == 0 then
       ngx.status = ngx.HTTP_BAD_GATEWAY
       ngx.exit(ngx.HTTP_OK)
     end
 
-    ngx.ctx.apicast = servers
+    ngx.ctx.upstream = servers
```

Comment:
😮  If I understand it correctly, this will choose where to send the request to the different container/services? !?? https://github.com/agaoglu/lua-upstream-nginx-module/blob/master/README.md#set_peer_down (I can

based on `server.query == 'system-provider' | 'system-developer' | 'apicast-staging' | `apicast-production` | 'backend'`

So powerful ... 👏

Reply:
By the way backend is not there I think it is another route right?

Reply:
Well it was already deciding between `apicast-staging` and `apicast-production`. So I just expanded the list. And `backend` is not there!